tests/fast-integration/monitoring/grafana.js

```diff
@@ -93,35 +91,32 @@ async function manageSecret(action) {
     kind: 'Secret',
```

Comment:
Could you please refactor this function (split into `createProxySecret`, `deleteProxySecret`)?